module/Decision/src/Model/Member.php

```diff
@@ -730,4 +730,33 @@ public function clearLists()
     {
         $this->lists = new ArrayCollection();
     }
+
+    /**
+     * Returns true the member is currently installed as a board member and false otherwise.
+     *
+     * @return bool
+     */
+    public function isBoardMember(): bool
+    {
+        foreach ($this->getBoardInstallations() as $boardInstall) {
+            if ($this->isCurrentBoard($boardInstall)) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * Check if this is a current board member.
+     *
+     * @return bool
+     */
+    protected function isCurrentBoard(BoardMember $boardMember): bool
+    {
+        $now = new DateTime();
+
+        return $boardMember->getInstallDate() <= $now
+            && (null === $boardMember->getDischargeDate() || $boardMember->getDischargeDate() >= $now);
```

Comment:
P.S. this is an invalid assumption. The board is often discharged 3-6 months after they stop being board. It is probably desirable to remove website admin rights earlier.

Reply:
This is the initial definition of the `IsBoardMember` assertion ([is still the same since its creation in 2015](https://github.com/kokx/gewisweb/commit/4d688bb96e04bf9e1d4f7f3c30cf3fa56ebc5d05#diff-084dcf54fe33d2065303f60e660218e039d61b3bef794b1a3413b91290356d18)).

I think we have had this discussion in the past and came to the conclusion that it is useful for the old board to still have the admin rights, but maybe I am not remembering this correctly.

Reply:
I think we discussed this within the committee at some point. We didn't change it following that. So I don't see why we would change it now.